nasa/earthdata-search
# EDSC-3140: Adding CSDA badge to collection results

# Overview

### What is the feature?

Adds the CSDA badge to collection search results

### What is the Solution?

Adds the CSDA badge to collection search results

### What areas of the application does this impact?

Search results

# Testing

### Reproduction steps

- **Environment for testing:** UAT
- **Collection to test with:** Any CSDA collection result

1. Search for CSDA collections
2. Confirm the badge shows for each CSDA collection
3. Confirm a tooltip shows when hovering the badge

# Checklist
